ceph/ceph.py

```diff
@@ -1097,6 +1097,58 @@ def get_nodes_in_location(self, location: str) -> list:
         """Return the list of nodes found in the location."""
         return [node for node in self.node_list if node.vm_node.location == location]
 
+    def get_node_ip_by_hostname(self, hostname):
+        """
+
+        Args:
+            hostname:
+
+        Returns:
+
+        """
+        cmd = "hostname -I | awk '{print $1}'"
+        node_list = [node for node in self.get_nodes() if node.hostname == hostname]
+        node = node_list[0] if len(node_list) > 0 else None
+        return node.exec_command(cmd=cmd)[0].replace("\n", "")
+
+    def get_node_ips_by_role(self, role):
+        """
+
+        Args:
+            role:
+
+        Returns:
+
```

Comment:
@manasagowri Can we please add missing doc?

Reply:
Done.